src/draw/lv_img_cache.c

```diff
@@ -200,16 +193,95 @@ void lv_img_cache_invalidate_src(const void * src)
  *   STATIC FUNCTIONS
  **********************/
 
+static bool find_entry(const lv_img_src_t * src, const lv_color_t * color, lv_img_dec_ctx_t * dec_ctx,
+                       lv_img_cache_entry_t ** out)
+{
+    /*Is the image cached?*/
+    lv_img_cache_entry_t * cached_src = NULL;
+
 #if LV_IMG_CACHE_DEF_SIZE
-static bool lv_img_cache_match(const void * src1, const void * src2)
+    if(entry_cnt == 0) {
+        LV_LOG_WARN("lv_img_cache_open: the cache size is 0");
```

Comment:
```suggestion
        LV_LOG_WARN("the cache size is 0");
```
 
The function name is added to the log automatically.